lenses/tests/test_cockpit.aug

```diff
@@ -0,0 +1,94 @@
+module Test_cockpit =
+(************************************************************************
+ * Basic Parse
+ *************************************************************************)
+
+   let conf = "
+[WebService]
+Origins = https://somedomain1.com https://somedomain2.com:9090
+
+MaxStartups= 10:30:60
+[Log]
+Fatal =criticals warnings
+
+[Session]
+
+IdleTimeout=15
+"
+
+   test Cockpit.lns get conf =
+  {  }
+  { "WebService"
+    { "Origins" = "https://somedomain1.com" }
+    { "Origins" = "https://somedomain2.com:9090" }
+    {  }
+    { "MaxStartups" = "10:30:60" }
+  }
+  { "Log"
+    { "Fatal" = "criticals" }
+    { "Fatal" = "warnings" }
+    {  }
+  }
+  { "Session"
+    {  }
+    { "IdleTimeout" = "15" }
+  }
+
+(************************************************************************
+ * Insert simple
+ *************************************************************************)
+    test Cockpit.lns put conf after
+       set "Log/EXAMPLE" "text"
+    = "
+[WebService]
+Origins = https://somedomain1.com https://somedomain2.com:9090
+
+MaxStartups= 10:30:60
+[Log]
+Fatal =criticals warnings
+
+EXAMPLE=text
+[Session]
+
+IdleTimeout=15
+"
+
+
+(************************************************************************
+ * Insert Origin makes sense
+ *************************************************************************)
+
+    test Cockpit.lns put conf after
+       insa "Origins" "WebService/Origins[last()]";
+       set "WebService/Origins[last()]" "https://thirdhost.com:8080"
```

Comment:
Not sure `insa` is necessary here. You could simple use:

```suggestion
       set "WebService/Origins[.='https://thirdhost.com:8080']" "https://thirdhost.com:8080"
```

I think

Reply:
Ah no, my bad, it's because of `MaxStartups`.

Don't you want to use a `seq` inside the `Origins` entry instead? That would be easier to manipulate:

```
{ "WebService"
  { "Origins"
    { "1" = "https://somedomain1.com" }
    { "2" = "https://somedomain2.com:9090" } }
  { "MaxStartups" = "10:30:60" } }
```

Reply:
I wasn't sure what the preferred approach was on that front.

If you think a `seq` would be cleaner I can try to make that work... I'll have to re-read the doc on that operator.

Reply:
checking back in....

Reply:
Ping

Reply:
Do you want to keep this lens as-is, or modify it to use a `seq` ?

I can't see an obvious advantage to one approach over the other

Reply:
I'd probably need a hand converting to a `seq`.  Mostly due to it being a bit since I've played with augeas.

Reply:
Sorry I could not be of help here.
I still think it's an abitrary change, and most lenses that I can find seem to use the same approach ie.
```
/files/etc/cockpit/cockpit.conf/WebService/Origins[1] = "https://somedomain1.com"
/files/etc/cockpit/cockpit.conf/WebService/Origins[2] = "https://somedomain2.com:9090"
```